drafts/prometheus-alertmanager-rhocp-open-liberty.adoc

```diff
@@ -0,0 +1,207 @@
+```
++
+. Once your pods are up and running, expose the route:
++
+```
+❯ oc expose svc/alertmanager-operated -n prometheus-operator
+route.route.openshift.io/alertmanager-operated exposed
+❯ oc get route -n prometheus-operator
+NAME                    HOST/PORT                                                                  PATH   SERVICES                PORT   TERMINATION   WILDCARD
+alertmanager-operated   alertmanager-operated-prometheus-operator.apps.jenniferc.os.fyre.ibm.com          alertmanager-operated   web                  None
+prometheus-operated     prometheus-operated-prometheus-operator.apps.jenniferc.os.fyre.ibm.com            prometheus-operated     web                  None
+```
+
+You can now access the Alertmanager UI. Since we haven't configured Prometheus to send the Alertmanager any alerts yet, you won't see any alert groups at the moment. We'll do that next.
+
+=== Receiving alerts via Prometheus Alertmanager
+Now that the Alertmanager is set up, we need to configure Prometheus to talk to it.
+
+. First, we'll need to expose the Alertmanager port. On the RHOCP web console, click on Networking > Services > Create Service. Create a ClusterIP service:
++
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: alertmanager-example-service
+  namespace: prometheus-operator
+spec:
+  type: ClusterIP
+  ports:
+  - name: web
+    port: 9093
+    protocol: TCP
+    targetPort: web
+  selector:
+    alertmanager: alertmanager-main
+```
++
+The `selector.alertmanager` must match your Alertmanager's name, if you changed it from the default value.
++
+
+. On the RHOCP web console, click on Operators > Installed Operators > Prometheus Operator > Prometheus, then click on the name of your Prometheus instance.
+. In the YAML, add your new service to the `alertmanagers` for Prometheus to talk to.
++
+```
+spec:
+  alerting:
+    alertmanagers:
+      - name: alertmanager-example-service
+        namespace: alertmanager
+        port: web
+```
++
+. Go to your Prometheus route, then click on Alerts. Ensure that at least one alert is firing.
+. Verify that Alertmanager has received the alert by going to the Alertmanager route.
++
+image::/img/blog/prometheusAM_rhocp_alertmanager_alerts.png[Alert viewed on Alertmanager web UI, align="left"]
++
+. Check your Slack channel to see your message.
++
+image::/img/blog/prometheusAM_rhocp_slack_alert.png[Alert on Slack, align="left"]
+
+For more advanced configurations, check out the link:/blog/2020/01/29/alerts-slack-prometheus-alertmanager-open-liberty.html#tips[additional tips for when you’re creating larger alerting systems] in the original blog post. To load in new rules, you can edit your Prometheus Rule instance's YAML from the web console, under Operators > Installed Operators > Prometheus Operator > Prometheus Rule. Similarly, to update your Alertmanager configuration, simply edit your `alertmanager.yaml` and re-create the secret.
```

Comment:
Update
"To load in new rules, you can edit your Prometheus Rule instance's YAML from the web console, under Operators > Installed Operators > Prometheus Operator > Prometheus Rule. Similarly, to update your Alertmanager configuration, simply edit your `alertmanager.yaml` and re-create the secret."
To
"To load in new rules, you can edit your `PrometheusRule` instance's YAML from the web console, under Operators > Installed Operators > Prometheus Operator > Prometheus Rule. Similarly, to update your Alertmanager configuration, simply edit your `alertmanager.yaml` file and re-create the secret."